app/helpers/application_helper.rb

```diff
@@ -517,7 +517,12 @@ def current_url_params(permitted: [])
   end
 
   def app_metadata
-    { UISettings: ui_settings, version: SETTINGS[:version].short, docUrl: documentation_url }
+    {
+      UISettings: ui_settings, version: SETTINGS[:version].short, docUrl: documentation_url,
+      location: Location.current&.id,
+      organization: Organization.current&.id,
+      user: User.current&.id
```

Comment:
nit: How about `locationId`, `organizationId`, `userId` for the keys?

Reply:
Should we make these into objects that also contain the taxonomy label or the user full name so they can be used for display purposes?

Reply:
makes sense